components/bl-credit-card-component/README.md

```diff
@@ -0,0 +1,197 @@
zygD3/ADzzzzzzzzzzzzzzzzzzzzzzzzzzzz//xAAnEQEAAgECAgsBAAAAAAAAAAABABEhEDFBUSAwQGFxgZGhscHR4f/aAAgBAwEBPxDoJgHsqhGMJZKQbihoPZeJjcaN3h7wRLNKa0g309+uGga0YfwDn48ohZV4sPN+/wCIWTZufZyfmBVscng6bY+Gmzp7dDaOlkpGLzHkYPiUaoqDhe+ZSOuCsUm1eHKbzST0nOEHlh+46Vth49UAzZps0cXFs8HP8mB7GV3P4wpa6Y/Ty94z3lt84Y3G+TAfFw5lwLgV1F2m+PGhoLcNh+nu+IgkovtWHZyyznBD0A+YNd3mF/nf6QCCghxoePVYZjW+glKLxovSssgW9XUqVKlSpUqVExLQcAO3f//EACQRAQACAQMDBQEBAAAAAAAAAAEAERAhMUEgMFFAkaGxwdFh/9oACAECAQE/EOgTFHpQWEKypaVUBcb7MD6yJWLZtKrr251YJePFb5htFE9g/D+ywGnl/Zd9wxvh5xu69+HeGKJaEF4+9ZRJS8yh23n/AGGQ5gVuS/yGsMCiPjtFE3Y3YN5wV7TQ7r9I8Jp5QAGxBUOCo9Ilxai32KMbZv6HrdVvHDZdE+Wx4l/Aiq2x64fHa2zfSuWWsXlEVHbuXLly5cuXLg6ykTFv13//xAArEAEAAQIFAwMFAQEBAQAAAAABEQAhEDFBUWFxgaGRscEgMEDR8VDwYOH/2gAIAQEAAT8Q/wBVW+LopmLgiI9a/t1ATlm9UjxCoIoTeBf8pbmJuAfNgRcw/onzQlgAAaH+RJkH2iQINWUpMUV2l+6dueoyL74Aw6upLDsr++/dP8wriZIG+TUzikMskrtkr++/df337oLxDy2QLM7TgWGl9PQZXYpSfOTnoXesUZiTCYClIGcTgcKZBDqtJQnOQeqB7NQeQaQE3hPHekRwWkBWCjQ7IKdExu7TIkKW2ga/v/3RPl9/7oblgCboXXXEvvyCH/3iipqi4Okw+K/vv3TmyYQoM05y/dsiwycwVe36jUgwO4jyUusQLZPxjNGkiFozmgQua/KZ3GZjV0pqqTNLxwcFilCdiiioSDkXyWgjCSGFt3T3acOj4ydvAHXPnCX5Ah2S+FUKt5/I3yKCB5n4iO+fNNAhi0u5r7NeazxODgHvxSCL02SI9YcSo14W6zt0twh3x1WLfQ+6BcyE4gChhprYugecQJiT1xRBqstZkhJwJSh8iRrEckspnybiMkQhJlnFQIRuQZTvlSUW9AbDYCANgwN+WL6I+GiXIEmJ7f8AGg1LEKvAbBkGM2kIh3C+VUNMjBsxexwStDrJ3HcSyUdcIpJRt2HxFCkwyTkq+HEs/B2QnlMJxDB+N6w7TTUB50jKvVcFAuh1qXxM5uEP3t9GDbJ+MY5kC7rTEXqGNsBLVugsxkSPziXA4OZOi6vCXpSlBbGE8EPlQ4RSyDKRzrIm9NZgnoMShnGZFkvMnYvzpS56Qqkube9GgqMXC7u6dY2xysOVyiIdRKcM7cqpMjHpSGEHaBhPXBloZDovxSASRJGr0YhDmEJ0k9mIcnhHghLOY7UspDEDoH3oeZ4elPM4IEXKwc0Q4E0ciPtjFLC3wx8iggDbBmMnEg3foF9KCFHmq1TVN1xUCYjgFgJwBTICjvZikkNBoTBwB2GMFd+OOBaHOAtmo6QdsDKNyoe/QIEdZEpz5GZ1NhwEHbB86LRRfNR0S6UMFgCADIPvKDAKuwUwsq3qvzg9RezZM+PosqwTcK/biYRWqZLYO3qfROBHVAn5HCdhRszcoOy0WtLUiEiYPda1ABK1foA3MlA9AwUAkeADDymEGycMfoaFG68HqexYdl6VovcxeZ4nl++mrthuQV4RhmHwMVI+pRJQBz9wU6F+tI+XQCSALm2cJcmQcUt+SbeA+76EtoZgyFl+qO5iggjblYmbQVyypW0NgKS5ulmn6IlIVmW90sYw980mYZfrB2cLB27ksvUDvQySfRemsB8hhJdDf8A0JO+Aj701WbGoNhwkJw/Q9xEG4L7vBSJDOFle3BYFdKQdBG7nz9CfanEdTZMykgOsPqepqaPbCVyzrqV7KhfIDoA92MBJIdt/q6HxROTOAfPOBqmEZIkJQ/Mmci53g4llESVkAdVqCYQ3UOhkcB+CShhqwyivbQZZXMgIowd0BletCOSPTA5kC5E6YAtC3nVj4pyHrUIiaG4D3oS4ANgIPpCa9ytvDRo4guWF6r1J7UkoVBDAPcNQm2BORXii42gCV7U66XbL8PVjvQr7hN9wav0TfJBGieQONxJBiznhMdeH4a0rQhI0ie84a9wmhAlN5UUEowIgAMjBm0KlFXNwBircPAXEQsn2YTkXlgQT2KC42UfUPpsFAGDjl4LtQgYZpkp5VV1jCIbG0TmuAleCoOASxd5pysr1/DQK4uosNRZoTlzTKLASIumaRmPpQLWGbIybEKjtTSgvAQkJfmmxLVq7BEBuTpU3FgISFBOtkypQKoBmtLPmADAyJPI1UAxVBvCJyyLfZ+hvZQnEoCZ3EjfSpfGTbCFqbWueGn/chwF1VyKLkmEAslE31X0zpaToc1IR3FAnxf1X8b+q1Oyw9A2O2DAStqRuhjuubNLPTqfxLfSNSQ2N9O2tTnAojI0ImZg3WRuhzklJzxU0ZShLJKcXwdpqUkCjclYrLD2kfMQVMkyt70OJ+XKbI3zEQOZSAtAap3ukgWvaylHAYHMwbmbGW9r3omS6zGtFTEEMzIZpd1CzMJyAgZCQWRncXMDmUu2GN71OhGVCiJLDGWWXICBI45aDLKaOQDJBirGR6FCJsr29pbQhct4DpfS6WACBhim3WuYuC9kuXRkikzCoNZgSCSBShGCCJgWDEjaAuxfddks6gtEgargXjZSRBBNy74+iQqy5GwF1oEbIHlUuf8edqCCD8eyVnNOXu+rv+BKO2eVgyERL64vXu/nQvc9lHkWwpJ3ZdCD/ANR//9k=";
```

Comment:
instead of the huge string, I would add a short comment string which describes it is a base 64 image